packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -366,7 +366,7 @@ function DetailsModal(props: {
               formatNumber(Number(balanceQuery.data.displayValue), 5)
             ) : (
               <Skeleton height="1em" width="100px" />
-            )}
+            )}{" "}
```

Comment:
u can string concat the value + symbol and show the skeleton for both (probably better anyways)